# Fehlerbehandlung bei Callbacks (alt)
ðŸ’¥
- Statt mit `do-catch` arbeitet man mit dem Result Enum:

```swift
func fetchInbox(completion: @escaping (Result<[Message], Error>) -> Void)
```

## Â Zusammenfassung
- Wie hat man frÃ¼her bei Callbacks das Error-Handling implementiert?

#learning unit#